test/test_util.py

```diff
@@ -48,6 +48,7 @@
 # numbers used for timeouts
```

Comment:
That's still unrelated to ensuring a pool manager is closed, but I'm less averse to it being included here for expediency. It also isn't explained in any of the commits or their messages (all the commits are a single line according to the GitHub UI).

So future developers will find this and have no clear understanding of the *why* unless they very closely read the diff. It should at least have been split into its own commit for documenting in our history.